docs/api-site-config.md

```diff
@@ -94,6 +94,11 @@ Information for Algolia search integration. If this field is excluded, the searc
 - `indexName` - the Algolia provided index name for your search (usually this is the project name)
 - `appId` - Algolia provides a default scraper for your docs. If you provide your own, you will probably get this id from them.
 
+#### `anonymizeIP` [boolean]
```

Comment:
It must be documented [here](https://github.com/facebook/docusaurus/blob/master/website/docs/using-plugins.md#docusaurusplugin-google-analytics).

Reply:
It is...

Reply:
Hey @RDIL it should also be documented on the plugin's README file as pointed out by Alexey.

Reply:
Fixed!